### PR TITLE
ENG-3672 add keycloak service

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: src/services/initializer-service.ts [hunk 1]
```diff
@@ -100,6 +110,9 @@ export class InitializerService {
     this.filesys.createSubDirectoryIfNotExist(CONFIG_FOLDER)
     this.filesys.createSubDirectoryIfNotExist(...OUTPUT_FOLDER)
     this.filesys.createSubDirectoryIfNotExist(SVC_FOLDER)
+    this.filesys.createSubDirectoryIfNotExist(...KEYCLOAK_FOLDER)
```
Comment (cecchisandrone) [inline src/services/initializer-service.ts:117]: better creating a single function to group keycloak folders creation

File: src/services/initializer-service.ts [hunk 2]
```diff
@@ -147,6 +171,33 @@ export class InitializerService {
         templateVariables
       )
     }
+
+    const keycloakRealm = path.join(
```
Comment (cecchisandrone) [inline src/services/initializer-service.ts:183]: group keycloak stuff in a function for better readability

File: package.json [hunk 3]
```diff
@@ -104,5 +104,10 @@
     "ts-node": "10.7.0",
     "tslib": "2.3.1",
     "typescript": "4.6.3"
+  },
+  "org_entando_dependencies": {
+    "mysql": "mysql:8.0.26",
```
Comment (cecchisandrone) [inline package.json]: put here only the version not the full image name

Comment (alepintus) [reply package.json]: done

File: src/services/initializer-service.ts (hunk 2 above)
Comment (alepintus) [reply src/services/initializer-service.ts:183]: done

File: src/services/initializer-service.ts (hunk 1 above)
Comment (alepintus) [reply src/services/initializer-service.ts:117]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**